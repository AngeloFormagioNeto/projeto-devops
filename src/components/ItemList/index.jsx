import React from "react";
import './styles.css';

function ItemList({title, description}) {
  const link = `https://github.com/AngeloFormagio/${title}`;
  return (
    <div className="item-list">
      <strong><a target="blank" href={link}>{title}</a></strong>
      <p>{description}</p>
      <hr />
    </div>
  );
}

export default ItemList;
