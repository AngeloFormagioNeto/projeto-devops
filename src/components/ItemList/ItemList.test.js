import React from 'react';
import renderer from 'react-test-renderer';
import ItemList from './index';

test('ItemList snapshot matches', () => {
  const tree = renderer
    .create(<ItemList title="Test Repo" description="Test description" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});