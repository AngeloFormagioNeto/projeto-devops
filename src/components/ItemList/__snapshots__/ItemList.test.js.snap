// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemList snapshot matches 1`] = `
<div
  className="item-list"
>
  <strong>
    <a
      href="https://github.com/AngeloFormagio/Test Repo"
      target="blank"
    >
      Test Repo
    </a>
  </strong>
  <p>
    Test description
  </p>
  <hr />
</div>
`;
